node/src/chain_spec/turing.rs

```diff
@@ -16,10 +16,7 @@ use turing_runtime::{
 
 static TOKEN_SYMBOL: &str = "TUR";
 const SS_58_FORMAT: u32 = 51;
-const TOTAL_TOKENS: u128 = DOLLAR * 1_000_000_000;
 static RELAY_CHAIN: &str = "rococo-local";
-static TURING_RELAY_CHAIN: &str = "ksmcc3";
```

Comment:
added to per-chain spec basis since irsal mentioned that the relaychain and para id are different for staging and live. but i guess i can define separate for live

Reply:
fixed